pyaedt/edb_core/hfss.py

```diff
@@ -441,80 +448,31 @@ def create_coax_port_on_component(self, ref_des_list, net_list):
+                            self._active_layout,
+                            pin[1].pin.GetNet(),
+                            port_name,
+                            pin[1].pin,
+                            to_layer_pos,
+                        )
+                    ):
+                        coax.append(port_name)
         return coax
 
-    @pyaedt_function_handler()
-    def create_coax_port_on_component_per_pin(self, reference_designator, pin_number, port_name=""):
-        """Create a coaxial port on a component per pin.
-
-        Parameters
-        ----------
-        reference_designator : str
-            Reference designator of the component.
-        pin_number : str
-            Pin number.
-        port_name : str, optional
-            Name of the net. The default is ``""``, in which case a name is automatically assigned.
-
-        Returns
-        -------
-        Port name when successful; ``False`` otherwise.
-        """
-        comp = self._pedb.core_components.components[reference_designator]
-        pin = comp.pins[pin_number]
-        edb_net = pin.pin.GetNet()
-        edb_pin = pin.pin
-        if is_ironpython:
-            _, from_layer, to_layer = edb_pin.GetLayerRange()
-        else:
-            _, from_layer, to_layer = edb_pin.GetLayerRange(None, None)
-
-        if from_layer == comp.placement_layer:
-            edb_layer = from_layer
-        else:
-            edb_layer = to_layer
-
-        net_name = pin.net_name
-
-        if not port_name:
-            port_name = "{}_{}_{}".format(reference_designator, net_name, pin.pin.GetName())
-
-        if self._edb.Cell.Terminal.PadstackInstanceTerminal.Create(
-            self._active_layout, edb_net, port_name, edb_pin, edb_layer
-        ):
-            return port_name
-        else:
-            return False
-
-    @pyaedt_function_handler()
-    def create_coax_port_on_component_per_net(self, reference_designator, net_name, port_name=""):
-        """Create a coaxial port on a component pin.
-
-        Parameters
-        ----------
-        reference_designator : str
-            Reference designator of the component.
-        net_name : str
-            Name of the net.
-        port_name : str, optional
-            Name of the net. The default is ``""``, in which case a name is automatically assigned.
-
-        Returns
-        -------
-        bool
-            `Port name when successful; ``False`` otherwise.
-        """
-        comp = self._pedb.core_components.components[reference_designator]
-        pin_number = ""
-        for pnum, pin in comp.pins.items():
-            if pin.net_name == net_name:
-                pin_number = pnum
-                break
-        return self.create_coax_port_on_component_per_pin(reference_designator, pin_number, port_name)
-
     @pyaedt_function_handler()
     def create_hfss_ports_on_padstack(self, pinpos, portname=None):
         """Create a HFSS port on a padstack.
```

Comment:
```suggestion
        """Create an HFSS port on a padstack.
```

Reply:
If the word following an "a" is a vowel sound (like HFSS), you use "an" rather than "a". This is one of the odd English rules.